Sources/ApolloCodegenLib/ApolloCodegenOptions.swift

```diff
@@ -52,6 +52,16 @@ public struct ApolloCodegenOptions {
     }
   }
   
+  /// Enum to select how to handle properties using a custom scalar from the schema.
+  public enum CustomScalarFormat: Equatable {
+    /// Uses a default type instead of a custom scalar.
+    case `default`
```

Comment:
I would call this `none` rather than `default`, since it's applying no formatting to custom scalars.